DesignSafe-CI/portal
# DES-2302: NCO TTC Grants page

## Overview: ##
Adding new NCO TTC Grants page

## PR Status: ##


## Related Jira tickets: ##

* [DES-2302](https://jira.tacc.utexas.edu/browse/DES-2302)
* [DES-2348](https://jira.tacc.utexas.edu/browse/DES-2348)

## Summary of Changes: ##
Moved previous NCO Scheduler page to url /nco/scheduler
Added new NCO TTC Grants page at url /nco/ttc_grants

## Testing Steps: ##
1. Make sure you have the NCO settings in your local designsafe.env (contact me for them)
2. Go to /nco/scheduler and make sure it looks like production's /nco page
3. Go to /nco/ttc_grants and make sure there are items
4. on the TTC Grants page, make sure the filters dropdown loads and has items.
5. test that the filters and sort options change the results (sometimes it might not be obvious if you're going between something like Start Date Ascending and End Date Ascending)

## UI Photos:

## Notes: ##
